### PR TITLE
Rhea annotations

### PR DESCRIPTION
**Pull request by pecholleyc**
### Main improvements in this PR:
 A total of **590** Human-GEM reactions were associated Rhea ID, and the corresponding master Rhea id. More information about the mapping process to Rhea can be found [here](https://github.com/SysBioChalmers/Human-GEM/issues/224#issuecomment-797778447)
- 69 were mapped via gene id -> UniProt id -> MetaNetX rxn id -> Rhea id
- 487 were mapped via equation mapping
- 34 were mapped by both methods

**I hereby confirm that I have:**

- [X] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch

### CODE REVIEW

File: model/reactions.tsv [hunk 1]
```diff
@@ -6024,11 +6024,11 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "HMR_5984"	""	""	""	"r2069"	""	"r2069"	"MNXR105978"	"HMR_5984"	"RCR40526"	"2.A.3.8.15"	0	"MA-R05984"	""	""
 "HMR_5985"	""	""	""	"r2070"	""	"r2070"	"MNXR105979"	"HMR_5985"	"RCR40160"	"2.A.3.8.15"	0	"MA-R05985"	""	""
 "HMR_5986"	""	""	""	"r2071"	""	"r2071"	"MNXR105980"	"HMR_5986"	"RCR41449"	"2.A.3.8.15"	0	"MA-R05986"	""	""
-"HMR_5987"	""	"FE2t"	""	"r2072"	""	"FE2DMT1"	"MNXR99505"	"HMR_5987"	"RCR40527"	"2.A.55.2.3"	0	"MA-R05987"	""	""
+"HMR_5987"	""	"FE2t"	""	"r2072"	""	"FE2DMT1"	"MNXR99505"	"HMR_5987"	"RCR40527"	"2.A.55.2.3"	0	"MA-R05987"	"RHEA:29441;RHEA:29440;RHEA:29581;RHEA:29580"	"RHEA:29439;RHEA:29579"
```
Comment (pecholleyc) [inline model/reactions.tsv]: Here, the Rhea master IDs can be difficult to associate programmatically to the Rhea IDs (2 master IDs versus to 4 Rhea IDs). Do you think the content should be changed, for instance by always having the same number of IDs in the two columns (this implies to duplicate Rhea master IDs)?

File: model/reactions.tsv [hunk 2]
```diff
@@ -8426,7 +8426,7 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "FAS80COA_L"	""	"FAS80COA_L"	""	""	""	"FAS80COA_L"	"MNXR99415"	""	""	""	0	"MA-R01036"	""	""
 "FBA5"	""	"FBA5"	""	""	""	"FBA5"	"MNXR99463"	""	""	""	0	"MA-R01038"	""	""
 "FCLTm"	""	"FCLTm"	""	""	""	"FCLTm"	"MNXR99471"	""	""	""	0	"MA-R01044"	"RHEA:22584"	"RHEA:22584"
-"FE2tm"	""	"FE2tm"	""	""	""	"FE2tm"	"MNXR99501"	""	""	""	0	"MA-R01046"	""	""
+"FE2tm"	""	"FE2tm"	""	""	""	"FE2tm"	"MNXR99501"	""	""	""	0	"MA-R01046"	"RHEA:29441;RHEA:29440;RHEA:29581;RHEA:29580"	"RHEA:29439;RHEA:29579"
```
Comment (pecholleyc) [inline model/reactions.tsv]: Same problem for that reaction.

File: model/reactions.tsv (hunk 1 above)
Comment (pecholleyc) [reply model/reactions.tsv]: Another solution would be to manually curate the association now, some Rhea ID can be probably removed.

Comment (Hao-Chalmers) [reply model/reactions.tsv]: Among the four Rhea ids mapped to `HMR_5987`, only `RHEA:29581` appears to be the correct association while the others can be ruled out according the reaction direction or metabolite compartment.

**Comment by Hao-Chalmers** [comment]
The most recent commit introduced some new associations that include one-to-multiple links between **Human-GEM** and **Rhea**. This is completely fine if the links can be verified programmatically or manually. Otherwise, they probably should be leave out. 

From the point of view of reviewer, it's alright to have less associations as long as they are accurate.

**Comment by mihai-sysbio** [comment]
One-to-many links increase the difficulty of using the annotations as they need to be treated differently in code. If possible, sticking to a single id simplifies the work.

**Comment by Hao-Chalmers** [comment]
@pecholleyc it's pleasant to go through these Rhea associations, especially the approach via equation mapping that nicely link to the ones with proper reaction direction. While the way through UniProt link to Rhea ids in undefined directions. Could the latter be revised with proper direction by using the equation mapping method?

**Review by Hao-Chalmers — APPROVED** [review]

**Comment by Hao-Chalmers** [comment]
@pecholleyc were these Rhea IDs with directed reaction updated manually, without the help of equation mapping?

**Pull request merged**